tardis/io/decay.py

```diff
@@ -62,7 +62,6 @@ def to_materials(self):
         -------
             : list
```

Comment:
```suggestion
        -------
        list
```
See section-5: https://numpydoc.readthedocs.io/en/latest/format.html#sections